### DISCUSSION TITLE
VM build image for VitrualBox

### DISCUSSION THREAD

**Discussion by artk42** [Start9Labs/start-os#421] [Ideas]
For evaluation purposes that would be great to have build for VM to give a try on home desktop, before messing up with Raspberry Pi

**Comment by kn0wmad** [comment]
> For evaluation purposes that would be great to have build for VM to give a try on home desktop, before messing up with Raspberry Pi

We have heard this several times and will be opening up to more architectures in the future.  For now it is theoretically possible, we just do not support it at this time.  You may find this helpful, https://raspberrytips.com/run-raspberry-in-virtual-machine/, as well as joining our community dev channel on Matrix: #community-dev:matrix.privacy34kn4ez3y3nijweec6w4g54i3g54sdv7r5mr6soma3w4begyd.onion
 for Tor, or #community-dev:matrix.start9labs.com for clearnet.  Thanks for reaching out!

**Reply by artk42** [reply]
Sure, that's it, I'm not talking about more archs, but just a VBox image of what you already have for a "several clicks to evaluate" experience.

**Reply by kn0wmad** [reply]
We have some additional ideas around being able to 'try out' EOS before using, but I will add this to my project list after we get v0.3.0 released.  Thanks for the feedback :)

**Reply by kn0wmad** [reply]
> Sure, that's it, I'm not talking about more archs, but just a VBox image of what you already have for a "several clicks to evaluate" experience.

Just to clarify, it is not possible to run the ARM architectures in Virtualbox; this is why we cannot make a VBox image at this time.  It may be possible using QEMU, you can find directions in the link I gave before.

**Comment by kn0wmad** [comment]
Hi there @artk42  - just wanted to let you know that you can now run eOS (as of today's release of v0.3.3) on a VM :)

Good luck on your sovereign journey!

**Reply by artk42** [reply]
Is there a specific link where i can find it? It looks like i dropped out and hardly can realize where to look for

**Reply by k0gen** [reply]
It's in the [releases](https://github.com/Start9Labs/embassy-os/releases).
